packages/cli/CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 <!-- ## Unreleased -->
 <!-- Add new, unreleased changes here. -->
 
+## v1.9.2 [12-06-2018]
+* Updated dependecies.
```

Comment:
"dependencies"

but could you actually summarize the relevant changelog items here since otherwise there's no information about what changed

Reply:
Updated. PTAL